.github/workflows/ci_test-full.yml

```
@@ -40,8 +40,8 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
-    - name: Weekly reset caching
-      run: echo "::set-output name=period::$(python -c 'import time ; days = time.time() / 60 / 60 / 24 ; print(int(days / 7))' 2>&1)"
+    - name: Reset caching
+      run: echo "::set-output name=period::$(python -c 'import time ; days = time.time() / 60 / 60 / 24 ; print(int(days / 2))' 2>&1)"
```

```suggestion
      run: echo "::set-output name=period::$(python -c 'import time ; days = time.time() / 60 / 60 / 24 ; print(f'd{int(days / 2) * 2}')' 2>&1)"
```
to keep in the name it is a day so it shall be `d2, d4, d6, ...`